cmd/new.go

```diff
@@ -135,19 +135,20 @@ var (
 	newBasicTemplate = `package main
 
 import (
-    "log"
+	"github.com/gofiber/fiber/v2/middleware/logger"
 
-    "github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2"
 )
 
 func main() {
-    app := fiber.New()
+	app := fiber.New()
+	app.Use(logger.New())
```

Comment:
This does add overhead, probably why the default server doesnt have it

Reply:
I think is useful to have it, maybe i can add a check for a env like "DEBUG" and activate only when the enviroment is develop?